samples/bluetooth/direct_test_mode/README.rst

```diff
@@ -143,13 +143,12 @@ Vendor specific commands can be divided into different categories as follows:
 * If the Length field is set to ``1`` (symbol ``CARRIER_TEST_STUDIO``), this field value is used by the nRFgo Studio to indicate that an unmodulated carrier is turned on at the channel.
   It remains turned on until a ``TEST_END`` or ``RESET`` command is issued.
 * If the Length field is set ``2`` (symbol ``SET_TX_POWER``), the Frequency field sets the TX power in dBm.
-  The valid TX power values are specified in the product specification ranging from -40 to +4.
-  0 dBm is the reset value.
+  The valid TX power values are specified in the product specification ranging from -40 to +4.0 dBm is the reset value.
```

Comment:
I think `0 dBm is the reset value.` belongs to a separate sentence.

Reply:
I think you are right.

Reply:
But I think you touched on an interesting case and this can be rewritten for clarity, for example:
```
The valid TX power values are specified in the product specification ranging from -40 to +4, where 0 dBm is the reset value.
```

Reply:
Even I was thinking of changing it as it was sounding quite confusing. Thanks!